tool/tctl/common/auth_command_test.go

```diff
@@ -107,14 +107,24 @@ func TestAuthSignKubeconfig(t *testing.T) {
 		wantError   string
 	}{
 		{
-			desc: "--proxy specified",
+			desc: "--proxy specified with URL scheme",
```

Comment:
nit: add tests as well for invalid cases. IE you get the expected error when using an invalid proxy address.

Reply:
Thanks for the review @knisbet. Added another test in a000f44dbaec56ad370826ebb47102917255bc47.